snyk-container-scan-policy-ts/README.md

```diff
@@ -0,0 +1,71 @@
+# snyn-container-scan-policy
+
+Scan Pulumi-managed Docker containers with Snyk and Pulumi Policy as Code:
```

Comment:
Suggest adding just a slight but of intro here that makes it clear this is an example that shows how to do this, rather than a tool _for doing_ this, if that makes sense.